src/indexer/flat_map_with_buffer.rs

```diff
@@ -0,0 +1,74 @@
+struct FlatMapWithgBuffer<T, F, Iter> {
```

Comment:
Did you really mean **g** in FlatMapWith**g**Buffer

Reply:
Updated the PR